README.md

```diff
@@ -0,0 +1,67 @@
+# uv-pre-commit
+
+[![uv](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/uv/main/assets/badge/v0.json)](https://github.com/astral-sh/uv)
+[![image](https://img.shields.io/pypi/v/uv.svg)](https://pypi.python.org/pypi/uv)
+[![image](https://img.shields.io/pypi/l/uv.svg)](https://pypi.python.org/pypi/uv)
+[![image](https://img.shields.io/pypi/pyversions/uv.svg)](https://pypi.python.org/pypi/uv)
+[![Actions status](https://github.com/astral-sh/uv-pre-commit/workflows/main/badge.svg)](https://github.com/astral-sh/uv-pre-commit/actions)
+[![Discord](https://img.shields.io/badge/Discord-%235865F2.svg?logo=discord&logoColor=white)](https://discord.gg/astral-sh)
+
+A [pre-commit](https://pre-commit.com/) hook for [uv](https://github.com/astral-sh/uv).
+
+Distributed as a standalone repository to enable installing Ruff via prebuilt wheels from
+[PyPI](https://pypi.org/project/uv/).
+
+### Using uv with pre-commit
+
+To run uv's compile via pre-commit, add the following to your `.pre-commit-config.yaml`:
+
+```yaml
+- repo: https://github.com/astral-sh/uv-pre-commit
+  # uv version.
+  rev: v0.1.23
```

Comment:
```suggestion
  # uv version.
  rev: v0.1.23
```
Maybe point/leave a note to https://github.com/astral-sh/uv/tags for available versions instead of specifying an out of date version or having to maintain the "latest one"?

Reply:
This is auto updated in `mirror.py`

Reply:
Yeah these are bumped on release already

Reply:
neat 🎉